js/components/pins-layer.js

```diff
@@ -6,20 +6,34 @@ import config from '../config'
 import '../../css/pins-layer.less'
 
 const getPinIcon = label => L.divIcon({
-  className: 'map-pin-icon',
+  className: `map-pin-icon ${!config.allowPinDrag && 'fixed-position'}`,
   html: `<div class="map-pin-content">${label}<div class='pin fa fa-map-pin' /></div>`
 })
 
+const allPins = []
```

Comment:
It's a bit strange it's a global/module variable. I guess this is something like cache, right? Do we need it at all? If so, I'd move it to the class as property, something like `.cachedPins`. Otherwise, we might have some weird issues if map is recreated (e.g. when we move between 2d and 3d views or we add mini-map in cross-section).